packages/web-components/src/components/card-group/__stories__/card-group.stories.ts

```diff
@@ -364,6 +369,7 @@ export default {
             : select('Grid mode:', gridModes, gridModes['Collapsed (1px)'], groupId);
         const offset = select('Offset:', ['0', '1'], '0', groupId);
         const cta = media ? '' : boolean('Add CTA card:', false, groupId);
+        const addCta = cardType === 'Card static' ? boolean('Add CTA Links:', false, groupId) : '';
```

Comment:
can we move this line below 
`        const tagGroup = cardType === 'Card - default' || cardType === 'Card static' ? boolean('Add tags:', false, groupId) : '';
`
in hopes that the knobs appears in order of the Card. (i.e. "Add media", "Add tags", "Add CTA links")